ReactWindows/ReactNative.Net46/Tracing/NullLoggingActivityBuilder.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace ReactNative.Tracing
+{
+    /// <summary>
+    /// Temporary NullLoggingActivityBuilder.
+    /// </summary>
+    public class NullLoggingActivityBuilder
+    {
+        /// <summary>
+        /// Dummy method to satisfy interface requirements.
+        /// </summary>
+        /// <returns>An empty disposable object.</returns>
+        public IDisposable Start()
+        {
+            return new NullDisposable();
+        }
+
+        /// <summary>
+        /// Empty disposable object.
+        /// </summary>
+        class NullDisposable : IDisposable
+        {
+            /// <summary>
+            /// Dispose method to conform to IDisposable.
```

Comment:
>IDisposable [](start = 45, length = 11)

You may remove this anyway, but you can make this a link using <see cref="IDisposable" />